Src/Test.UnitTests.Sarif.PatternMatcher/ReflectionTests.cs

```diff
@@ -55,5 +55,83 @@ public static void VerifyAllValidatorsExist(string definitionsFileName, Assembly
                 Assert.True(false, "Unexpected JSON structure in rules file");
             }
         }
+
+        public static void VerifyAllSharedStringsExist(string definitionsFilePath, string sharedStringsFilePath)
+        {
+            try
+            {
+                if (!File.Exists(definitionsFilePath) || !File.Exists(sharedStringsFilePath))
+                {
+                    return;
+                    // Should we Assert.True(false) here?
+                }
+
+                string content = File.ReadAllText(definitionsFilePath);
+
+                var jObject = (JObject)JToken.Parse(content);
+
+                IEnumerable<string> rules = jObject["Definitions"][0]["MatchExpressions"].Select(x => x["ContentsRegex"].ToString()).Distinct();
```

Comment:
>rules [](http://example.com/codeflow?start=36&length=5)

I feel like all this should have been in a helper, due to strong possibility of future reuse. and that by doing so you could break this apart, make it more readable/maintainable.